Adapt gsettings

Bring gsettings in line with the context of Debian.
Index: emcomm-tools-os-community/source/overlay/etc/skel/.bashrc
===================================================================
--- emcomm-tools-os-community.orig/source/overlay/etc/skel/.bashrc
+++ emcomm-tools-os-community/source/overlay/etc/skel/.bashrc
@@ -119,6 +119,3 @@ if ! shopt -oq posix; then
     . /etc/bash_completion
   fi
 fi
-
-# Set xterm colors for the default terminal profile
-/opt/emcomm-tools/bin/et-term
Index: emcomm-tools-os-community/source/overlay/usr/share/glib-2.0/schemas/90_ubuntu-settings.gschema.override
===================================================================
--- emcomm-tools-os-community.orig/source/overlay/usr/share/glib-2.0/schemas/90_ubuntu-settings.gschema.override
+++ emcomm-tools-os-community/source/overlay/usr/share/glib-2.0/schemas/90_ubuntu-settings.gschema.override
@@ -8,7 +8,7 @@ no-show-notifications = true
 show-desktop-icons = false
 
 [org.gnome.shell]
-favorite-apps = ['emcomm-tools.desktop', 'js8call.desktop', 'fldigi.desktop', 'et-predict-app.desktop', 'et-aircraft.desktop', 'org.gnome.Terminal.desktop', 'navit.desktop', 'min.desktop', 'org.gnome.Nautilus.desktop']
+favorite-apps = ['emcomm-tools.desktop', 'js8call.desktop', 'fldigi.desktop', 'et-predict-app.desktop', 'et-aircraft.desktop', 'org.gnome.Console.desktop', 'navit.desktop', 'org.gnome.Epiphany.desktop', 'org.gnome.Nautilus.desktop']
 last-selected-power-profile = 'performance'
 
 [org.gnome.system.location]
